server/graphql/v2/collection/AccountCollection.js

```diff
@@ -13,6 +16,29 @@ const AccountCollection = new GraphQLObjectType({
       nodes: {
         type: new GraphQLList(Account),
       },
+      stats: {
+        type: new GraphQLNonNull(AccountCollectionStats),
+        args: {
+          ...TimeSeriesArgs,
+          includeChildren: { type: GraphQLBoolean, defaultValue: false },
+        },
+        async resolve(collection, args) {
+          const dateFrom = args.dateFrom ? moment(args.dateFrom) : null;
+          const dateTo = args.dateTo ? moment(args.dateTo) : null;
+          const timeUnit = args.timeUnit || getTimeUnit(getNumberOfDays(dateFrom, dateTo, {}) || 1);
+
+          const collectiveIds = collection.nodes.map(c => c.id);
+
+          if (args.includeChildren) {
+            const childCollectiveIds = await Promise.all(
+              collection.nodes.map(async c => await c.getChildren({ attributes: ['id'] })),
+            );
+            collectiveIds.push(...childCollectiveIds.flat().map(c => c.id));
```

Comment:
Fetching 100 accounts with `includeChildren` will trigger 100 SQL queries just to fetch the list of children IDs. Can we improve that?